libbeat/docs/outputconfig.asciidoc

```diff
@@ -717,6 +717,8 @@ The index root name to write events to. The default is the Beat name. For
 example +"{beatname_lc}"+ generates +"[{beatname_lc}-]{version}-YYYY.MM.DD"+
 indices (for example, +"{beatname_lc}-{version}-2017.04.26"+).
 
+NOTE: This parameter's value will be assigned to the `metadata.beat` field. It can then be reused in Logstash's output section as `%{[@metadata][beat]}`
```

Comment:
Hello,
I agree about the accessed vs review wording.
About the logstash / beat output section confusion, considering this documentation section is about beats, I find the confusion would be unlikely but this would make sense to add a sentence explaining this indeed.

Thanks for reviewing my submission :)